server/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "hp-admin-crypto-server"
+version = "0.1.0"
+authors = ["PJ <pj@imagine-nyc.com>"]
+edition = "2018"
+
+[dependencies]
+hpos-state-core = { git = "https://github.com/Holo-Host/hpos-state", branch = "develop" }
+hyper = "0.12"
+futures = "0.1.25"
+serde_json = "1.0.39"
+lazy_static = "1.2"
+ed25519-dalek = { version = "1.0.0-pre.1", features = ["nightly", "serde"] }
```

```suggestion
ed25519-dalek = { version = "1.0.0-pre.2", features = ["nightly", "serde"] }
```
We've moved on to pre.2 in other repos...

---

fixed